app/plugin.go

```diff
@@ -769,7 +769,15 @@ func (a *App) processPrepackagedPlugins(pluginsDir string) []*plugin.Prepackaged
 	var wg sync.WaitGroup
 	for _, pluginPaths := range pluginSignaturePathMap {
 		wg.Add(1)
-		go a.processPrepackagedPlugin(pluginPaths, prepackagedPlugins, &wg)
+		go func(pluginPath *pluginSignaturePath) {
```

Comment:
nit: should `pluginPath` be named `pluginPaths` for consistency with the outer variable being passed in?